src/sections/Listings/LIstings.tsx

```diff
@@ -40,7 +40,7 @@ export const Listings: React.FC<ListingsProps> = ({ title }) => {
     );
   }
   return (
-    <div className='p-4'>
+    <div className='p-4 dark:bg-neutral-500'>
```

Comment:
Where do you get theme context from?

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/934af7151ebc2a241038463b34aa6250119bc21f